### PR TITLE
feat: add support for enabling pprof handlers

### PR DESCRIPTION
**Pull request by fiam**
Handlers are disabled by default and must enabled at compile time.
When compiled in, the handlers are run by default on port 6060 but
can be disabled via command line.

## Motivation and Context

- [ ] Tests or benchmark included
- [ ] Documentation is changed or added on [https://app.gitbook.com/](https://app.gitbook.com/)
- [ ] PR title must follow [conventional-commit-standard](https://github.com/wundergraph/wundergraph/blob/main/CONTRIBUTING.md#conventional-commit-standard)

### CODE REVIEW

File: router/cmd/pprof.go
Comment (StarpTech) [inline router/cmd/pprof.go:1]: The design of "cmd" is to host only entry points. Can we move this to internals?

**Review by StarpTech — CHANGES_REQUESTED** [review]

File: router/cmd/pprof.go
Comment (fiam) [reply router/cmd/pprof.go:1]: Sounds good. Should I move the rest of the code related to profiling into an internal package too?

Comment (StarpTech) [reply router/cmd/pprof.go:1]: Yes, please.

Comment (fiam) [reply router/cmd/pprof.go:1]: I've moved all code for profiling to `internal`, PTAL

Comment (StarpTech) [reply router/cmd/pprof.go:1]: Nice, ty.

**Review by StarpTech — APPROVED** [review]
LGTM

**Pull request merged**